templates/index.html.tt2

```diff
@@ -143,6 +143,14 @@
 				<div style="margin-left:auto">[% c.lh("Go to Page:") %]
 					<select class="favtag-btn" id="page-select">
 					</select>
+					<label for="columnCount">[% c.lh("Select Columns:") %]</label>
```

Comment:
I think this should only be visible in compact mode, it's confusing seeing it in the thumbnail view where it effectively does nothing.

Reply:
(I also wonder if a tooltip wouldn't be useful here -- This is technically only the number of custom columns and doesn't include title/tags, but it'd be way too long to write that in the main UI. Would also be a good place to note that changing this reloads the page)